crates/typst-layout/src/math/fragment.rs

```diff
@@ -227,14 +227,52 @@ impl From<FrameFragment> for MathFragment {
     }
 }
 
+/// A fragment of a glyph in math mode.
```

Comment:
```suggestion
/// A single glyph in math.
```
Currently it reads like this struct defines a part of a glyph, as opposed to a fragment in math which is a single glyph.

Reply:
Reworded!